docs/core/extensions/snippets/configuration/console-basic/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+using IHost host = Host.CreateDefaultBuilder(args).Build();
+
+// Ask the service provider for the configuration abstraction.
+IConfiguration config = host.Services.GetRequiredService<IConfiguration>();
+
+// Get values from the config given their key and their target type.
+int keyOneValue = config.GetValue<int>("KeyOne");
+bool keyTwoValue = config.GetValue<bool>("KeyTwo");
+string keyThreeNestedValue = config.GetValue<string>("KeyThree:Message");
+
+// Write the values to the console.
+Console.WriteLine($"KeyOne = {keyOneValue}");
+Console.WriteLine($"KeyTwo = {keyTwoValue}");
+Console.WriteLine($"KeyThree:Message = {keyThreeNestedValue}");
```

Comment:
Is it worth adding an example that binds to an object? I thought I'd need an `IOptions<T>` for that, but this works fine, I guess as long as `Microsoft.Extensions.Configuration.Binder` is installed.
```
public struct MyAppConfig {
    private string Name { get; init; }
}
...
config.GetValue<MyAppConfig>("MyApp")
```

Reply:
@Daniel15  What does all of your code look like? I'll add this but need to test it locally.